src/shared/interfaces.ts

```diff
@@ -1,7 +1,11 @@
+interface AnyObject {
```

Comment:
Not sure I'm a fan of this. The type itself is actually misleading, unless I'm misunderstanding something. Doesn't this define a type that is an object that can only have a key of string with a value of an empty object, `null` or `undefined`, which is essentially a data-less object?